File: src/main/java/com/realyoungk/sdi/dto/TelegramMessageDto.java
package com.realyoungk.sdi.dto;

import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
import com.fasterxml.jackson.annotation.JsonProperty;

@JsonIgnoreProperties(ignoreUnknown = true)
public record TelegramMessageDto(
        @JsonProperty("message_id") Long messageId,
        @JsonProperty("chat") TelegramChatDto chat,
        @JsonProperty("text") String text
) {
}

File: src/main/kotlin/com/realyoungk/sdi/dto/TelegramMessageDto.kt
package com.realyoungk.sdi.dto

import com.fasterxml.jackson.annotation.JsonIgnoreProperties
import com.fasterxml.jackson.annotation.JsonProperty

@JsonIgnoreProperties(ignoreUnknown = true)
data class TelegramMessageDto(
    @JsonProperty("message_id") val messageId: Long,
    @JsonProperty("chat") val chat: TelegramChatDto,
    @JsonProperty("text") val text: String
)